IDEAS/Buildings/Components/Zone.mo

```diff
@@ -6,46 +6,92 @@ model Zone "thermal building zone"
   extends IDEAS.Fluid.Interfaces.LumpedVolumeDeclarations(
     energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
     redeclare replaceable package Medium = IDEAS.Media.Air,
-    final mSenFac = corrCV);
+    final mSenFac = 1);
 
-  parameter Boolean allowFlowReversal=true
-    "= true to allow flow reversal in zone, false restricts to design direction (port_a -> port_b)."
-    annotation(Dialog(tab="Assumptions"));
-  parameter Boolean calculateViewFactor = false
-    "Explicit calculation of view factors: only for rectangular zones!"
-    annotation(Dialog(tab="Advanced"));
 
-  parameter Modelica.SIunits.Volume V "Total zone air volume";
+  parameter Modelica.SIunits.Volume V "Total zone air volume"
+    annotation(Dialog(group="Building physics"));
+  parameter Modelica.SIunits.Length hZone = 2.8
+    "Zone height: distance between floor and ceiling"
+    annotation(Dialog(group="Building physics"));
+  parameter Modelica.SIunits.Area A = V/hZone "Total conditioned floor area"
+    annotation(Dialog(group="Building physics"));
   parameter Real n50(min=0.01)=0.4
-    "n50 value cfr airtightness, i.e. the ACH at a pressure diffence of 50 Pa";
-  parameter Real corrCV=5 "Multiplication factor for the zone air capacity";
+    "n50 value cfr airtightness, i.e. the ACH at a pressure diffence of 50 Pa"
+    annotation(Dialog(group="Building physics"));
 
-  parameter Boolean linearise=true
-    "Linearized computation of long wave radiation";
+  parameter Boolean allowFlowReversal=true
+    "= true to allow flow reversal in zone, false restricts to design direction (port_a -> port_b)."
+    annotation(Dialog(tab="Advanced", group="Air model"));
+  parameter Real n50toAch=20 "Conversion fractor from n50 to Air Change Rate"
+    annotation(Dialog(tab="Advanced", group="Air model"));
 
+  parameter Boolean linearise=true
+    "Linearized computation of long wave radiation"
+    annotation(Dialog(tab="Advanced", group="Radiative heat exchange"));
+  parameter Boolean calculateViewFactor = false
+    "Explicit calculation of view factors: works well only for rectangular zones!"
+    annotation(Dialog(tab="Advanced", group="Radiative heat exchange"));
   final parameter Modelica.SIunits.Power QInf_design=1012*1.204*V/3600*n50/20*(273.15
```

Comment:
Actually, while scrolling to the end of the code i noticed the bug was in there already (as `n50toAch` was just moved to the top of the code, not introduced)